apps/veyfi/utils/constants.ts

```diff
@@ -0,0 +1,12 @@
+import {toAddress} from '@yearn-finance/web-lib/utils/address';
+
+import type {TAddress} from '@yearn-finance/web-lib/utils/address';
+import type {TWeeks} from '@yearn-finance/web-lib/utils/time';
+
+export const VEYFI_ADDRESS: TAddress = toAddress('0x90c1f9220d90d3966FbeE24045EDd73E1d588aD5');
+export const VEYFI_POSITION_HELPER_ADDRESS: TAddress = toAddress('0x5A70cD937bA3Daec8188E937E243fFa43d6ECbe8');
+export const YFI_ADDRESS: TAddress = toAddress('0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e');
+
+export const MAX_LOCK_TIME: TWeeks = 208;
```

Comment:
nit: @Majorfi We could export a type `IntRange` in the web-lib so that we can have a bit more control on these magical numbers

```typescript
type Enumerate<N extends number, Acc extends number[] = []> = Acc['length'] extends N ? Acc[number] : Enumerate<N, [...Acc, Acc['length']]>

type IntRange<F extends number, T extends number> = Exclude<Enumerate<T>, Enumerate<F>>
```
then we could use
```typescript
type TWeeks = IntRange<1, 52> // or whatever range we want
```